.github/workflows/component-bumper-release-0.42.yml

```diff
@@ -8,6 +8,7 @@ jobs:
 
   build:
     name: release-0.42 - CNAO Component Bump Job
+    if: (github.repository == 'kubevirt/cluster-network-addons-operator')
```

Comment:
Why do we have bump jobs at release branches?

Reply:
these are stable branches. 
The bumper also bumps stable branches, should a component gets updated. 
A good example is knmstate that got a new release on its stable branch, causing an auto bump on release-0.44 stable branch on CNAO..